### ISSUE TITLE
Keyboard Shortcuts

### ISSUE THREAD

**Issue by WinDroidGuy** [mitchmcdee/LifxController#15]
It would be nice to see these added, would make things a lot easier to toggle different lights.

**Comment by mitchmcdee** [comment]
A nice idea but a bit out of scope for now :) There's a way to add keyboard shortcuts to the menu in rumps, but only once you're in it-- and I think in this application the menus are simple enough to not need keyboard shortcuts once you've already navigate to them. But the idea of a global keyboard shortcut would be cool!
